thorlcr/graph/thgraph.hpp

```diff
@@ -475,8 +476,10 @@ class graph_decl CGraphElementBase : public CInterface, implements IInterface
     unsigned getOutputs() const { return outputs.ordinality(); }
     bool isSource() const { return isActivitySource(kind); }
     bool isSink() const { return sink; }
-    inline bool doLogging() const { return log; }
+    inline bool doLogging() const { return log || detailedTracing; }
```

Comment:
I think we should drop of doLogging/setLogging and replace with doDetailedLogging(),
but maybe in next PR..

Reply:
They are slightly different (see the distinction between logging and tracing, above).